aws/rust-runtime/aws-sigv4/src/lib.rs

```diff
@@ -59,14 +56,17 @@ impl<'a, S> SigningParams<'a, S> {
     pub fn service_name(&self) -> &str {
         self.service_name
     }
+
+    /// If the identity in params contains AWS credentials, return them. Otherwise, return `None`.
+    pub fn credentials(&self) -> Option<&Credentials> {
+        self.identity.data::<Credentials>()
+    }
```

Comment:
do we need this (or any!) public functions on signing params?

Reply:
I'd say no, but I'll just make the new fn `pub(crate)` for now